RELEASE_NOTES.md

```diff
@@ -27,6 +27,17 @@
 [`int8` or `int4` compressed key and value]: https://oneapi-src.github.io/oneDNN/v3.7/dev_guide_graph_sdpa_compressed_kv.html#doxid-dev-guide-graph-sdpa-compressed-kv
 [Gated Multi-Layer Perceptron (Gated MLP)]: https://oneapi-src.github.io/oneDNN/v3.7/dev_guide_graph_gated_mlp.html#doxid-dev-guide-graph-gated-mlp
 
+## AArch64-based Processors
+  * Add support for the Arm Compute Library (ACL) thread_local scheduler via ThreadpoolScheduler
+  * Improved memory efficiency in ACL matmuls by fixing a bug where scratchpad memory was not being used
+  * The following improvements along with numerous bug fixes are carried over from the 3.6.2 patch release:
```

Comment:
This section of release notes covers performance optimizations. Please Update the content accordingly.

Reply:
Quick explanation of top level sections:
* Performance optimizations covers functions that worked before, but run faster now without code changes on the user side.
* New functionality covers functionality that  was not available in the library before or not available in particular platform.
* Usability covers debugging, memory consumption, thread safety, and initialization costs.